modules/installation/pages/public-cloud-registration.adoc

```diff
@@ -0,0 +1,21 @@
+[[public-cloud-registration]]
+= Registration in the Public Clouds
+
+Since {productname} is preinstalled in Public Cloud images, there is no installation step required. However, {productname} Server needs to be registered with {scc} in order to receive updates.
```

Comment:
```suggestion
Because {productname} is preinstalled in Public Cloud images, there is no installation step required.
However, {productname} Server needs to be registered with {scc} to receive updates.
```